entity/person.rb

```diff
@@ -1,14 +1,15 @@
 require './interface/nameable'
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name, parent_permission: true)
     super
     @id = Random.rand(1..10_000)
     @name = name
     @age = age
     @permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] To implement the `has-many` side of the relationship(`association`) between person and rental, a person has many rentals, a method for adding rentals should be present in the Person class. Something like so: 

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Please kindly fix that.